kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -119,44 +117,61 @@
       LEARNER: () => UserKinds.LEARNER,
       COACH: () => UserKinds.COACH,
       ADMIN: () => UserKinds.ADMIN,
-      statusMessage() {
-        if (this.errorMessage) {
-          return this.errorMessage;
-        } else if (this.confirmationMessage) {
-          return this.confirmationMessage;
+      usernameAlreadyExists() {
+        const index = this.users.findIndex(user => user.username === this.username);
+        if (index === -1) {
```

Comment:
True. Will simplify. But I think I'll stick with findIndex since this is a pretty simple case.